modules/apps/object/object-web/src/main/resources/META-INF/resources/js/components/ObjectField/Tabs/BasicInfo/SearchableContainer.tsx

```diff
@@ -59,11 +59,14 @@ export function SearchableContainer({
 
 			<ClayForm.Group>
 				<Toggle
-					disabled={objectField.businessType === 'Encrypted'}
+					disabled={
+						objectField.businessType === 'Encrypted' ||
+						objectField.system
+					}
 					label={Liferay.Language.get('searchable')}
 					name="indexed"
 					onToggle={(indexed) => setValues({indexed})}
-					toggled={objectField.indexed}
+					toggled={objectField.indexed || objectField.system}
```

Comment:
@Artartur, the `indexed` property will define whether the object is searchable or not, there is no point in forcing it to be `toggled` if it is `system` only in the frontend if it is not passed to the backend. I confirmed with the backend folks here, and the only `system` field that makes sense to be seachebe is the `id` field, so the backend is right, the only thing we need to do in the frontend is disable the toggle when the field is system, you You can remove changes from this line.

Reply:
Sure, I got it, thx for explain, done!